cmd/relayproxy/service/gofeatureflag.go

```diff
@@ -198,6 +200,7 @@ func initDataExporter(c *config.ExporterConf) (ffclient.DataExporter, error) {
 	return dataExp, nil
 }
 
+// nolint: funlen
```

Comment:
Linter complains about a too-long function, but actually, the function is a huge `switch` expression so I'm not sure whether we should bother about it. I proposed to simply skip this check in that case. 

Hope to hear your opinion :blush: